CONTRIBUTING.md

```diff
@@ -182,7 +182,7 @@ Now we can build Microshift:
 ```sh
 make build
 CONFIG_ENV_ONLY=true ./install.sh
-sudo microshift run 
+sudo ./microshift run 
```

Comment:
so install.sh just downloads microshift in current dir?

Reply:
`install.sh` is only run here to configure the environment, the script exits before anything is actually downloaded; `./microshift` here is the binary generated from `make build`